### PR TITLE
Multiple services in bluechi status

### PR DESCRIPTION
**Pull request by ArtiomDivak**
This Pr will fix the bluechi status that did not work with multiple services in the status option. Also edited and aligned the printing for multiple services

### CODE REVIEW

File: src/client/method-status.c [hunk 1]
```diff
@@ -188,12 +188,15 @@ static void print_info_header(size_t name_col_width) {
...
-#define PRINT_AND_ALIGN(x)                                       \
-        do {                                                     \
-                fprintf(stdout, "| %s\t", unit_info->x);         \
-                if (!unit_info->x || strlen(unit_info->x) < 6) { \
-                        fprintf(stdout, "\t");                   \
-                }                                                \
+#define PRINT_AND_ALIGN(x)                               \
+        do {                                             \
+                fprintf(stdout, "| %s\t", unit_info->x); \
+                if (unit_info->x) {                      \
```
Comment (engelmi) [inline src/client/method-status.c]: Previously, we printed a `\t` when the field x is not defined/empty or its length was smaller than 6 chars. This changes the `or` to an `and`. Is it not a valid case that a field is empty, so we need to tab here?

File: src/client/method-status.c [hunk 2]
```diff
@@ -280,7 +287,7 @@ static int method_status_unit_on(Client *client, char *node_name, char **units,
                 int r = get_status_unit_on(client, node_name, units[i], max_name_len);
                 if (r < 0) {
                         fprintf(stderr,
-                                "Failed to get status of unit %s on node %s - %s",
+                                "Failed to get status of unit %s on node %s - %s\n",
```
Comment (mwperina) [inline src/client/method-status.c:290]: What's the new line for? Wouldn't it be better to have the whole message on a single line?

File: src/client/method-status.c [hunk 3]
```diff
@@ -208,10 +208,17 @@ static void print_unit_info(unit_info_t *unit_info, size_t name_col_width) {
...
         name_col_width += PRINT_TAB_SIZE;
-        for (i = PRINT_TAB_SIZE + name_col_width; i > PRINT_TAB_SIZE; i -= PRINT_TAB_SIZE) {
+        i = name_col_width;
+        while (i >= PRINT_TAB_SIZE) {
                 fprintf(stdout, "\t");
+                i -= PRINT_TAB_SIZE;
+        }
+        while (i > PRINT_TAB_SIZE) {
+                fprintf(stdout, " ");
+                i--;
```
Comment (mwperina) [inline src/client/method-status.c:218]: I'm just wondering if it wouldn't be better to extract table printing from https://github.com/eclipse-bluechi/bluechi/blob/main/src/client/method-list-units.c#L175 and reuse it also here. Calculate column lengths and using formatting string seems to me much easier to understand than this tab+space alignment

Comment (mwperina) [reply src/client/method-status.c:218]: But that can be done separately in a different PR

File: src/client/method-status.c (hunk 2 above)
Comment (ArtiomDivak) [reply src/client/method-status.c:290]: This one is not related to the issue.
But without this new line the output I get if the node is not connected to the controller is
`Failed to get status of unit virtnwfilterd.service on node fedora - Input/output erroradivak@fedora:~$`
so the new line will make it look like
`Failed to get status of unit virtnwfilterd.service on node fedora - Input/output error
adivak@fedora:~$`
I think it's look better and that why I added it.

File: src/client/method-status.c (hunk 3 above)
Comment (ArtiomDivak) [reply src/client/method-status.c:218]: If I will print only spaces without the tabs.
Will it be more understandable?

Comment (engelmi) [reply src/client/method-status.c:218]: Its probably fine as it is right now. 
As Martin wrote, we could optimize and reuse other code here, but thats a different task. Lets tackle it in a new PR. Could you create a GH issue for it if we want to track it? @mwperina

File: src/client/method-status.c (hunk 2 above)
Comment (mwperina) [reply src/client/method-status.c:290]: OK

**Review by engelmi — APPROVED** [review]
LGTM

**Pull request merged**